atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -208,19 +210,20 @@ def newTextPipeline(self, text: str, uuid: str):
-        competency_to_match = None
-        if competency: competency_to_match = competency[0]
-
-        properties = {
-                    "text_id": uuid,
-                    "text": text ,
-                    "competency_ids": [competency_to_match["properties"]["competency_id"]]
-        }
-        self.weaviate_client.add_embeddings(CollectionNames.TEXT.value, embedding, properties)
-        return competency_to_match["properties"]["competency_id"]
\ No newline at end of file
+        if not competency: return None
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)